torchao/quantization/quant_api.py

```diff
@@ -706,7 +708,6 @@ def _int8_dynamic_activation_int4_weight_transform(
     # weight settings
     block_size = (1, group_size)
     target_dtype = torch.int8
-    eps = torch.finfo(torch.float32).eps
```

Comment:
Good catch. I had to do the same in my PR, since the the quant primitives were using `torch.finfo(torch.bfloat16).eps` instead